config/prow/cluster/starter.yaml

```diff
@@ -3,24 +3,149 @@
 # their own files.
 ---
 apiVersion: v1
+kind: Namespace
+metadata:
+  name: prow
+---
+apiVersion: v1
 kind: ConfigMap
 metadata:
-  namespace: default
+  namespace: prow
   name: plugins
 data:
-  plugins.yaml: ""
+  plugins.yaml: |
+    plugins:
+      << your_github_org >>:
+      - approve
+      - assign
+      - blunderbuss
+      - cat
+      - dog
+      - help
+      - heart
+      - hold
+      - label
+      - lgtm
+      - trigger
+      - verify-owners
+      - wip
+      - yuks
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: prow
+  name: github-token
+stringData:
+  token: <<insert-token-here>>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: prow
+  name: hmac-token
+stringData:
+  # Generate via `openssl rand -hex 20`. This is the secret used in the GitHub webhook configuration
+  hmac: << insert-hmac-token-here >>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: prow
+  name: s3-credentials
+stringData:
+  service-account.json: |
+    {
+      "region": "minio",
+      "access_key": "<<CHANGE_ME_MINIO_ACCESS_KEY>>",
+      "endpoint": "minio.prow.svc.cluster.local",
+      "insecure": true,
+      "s3_force_path_style": true,
+      "secret_key": "<<CHANGE_ME_MINIO_SECRET_KEY>>"
+    }
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: test-pods
+  name: s3-credentials
+stringData:
+  service-account.json: |
+    {
+      "region": "minio",
+      "access_key": "<<CHANGE_ME_MINIO_ACCESS_KEY>>",
+      "endpoint": "minio.prow.svc.cluster.local",
+      "insecure": true,
+      "s3_force_path_style": true,
+      "secret_key": "<<CHANGE_ME_MINIO_SECRET_KEY>>"
+    }
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  namespace: default
+  namespace: prow
   name: config
 data:
   config.yaml: |
-    prowjob_namespace: default
+    prowjob_namespace: prow
     pod_namespace: test-pods
+
+    in_repo_config:
+      enabled:
+        "*": true
+
+    deck:
+     spyglass:
+       lenses:
```

Comment:
Just fyi. If we have a publicly accessible Minio URL this would also work:
gcs_browser_prefix: <minio-url>/minio/

But not sure if it's overly complicated for the starter.yaml

Reply:
In this sample the minio is not public